tests/unit/test_cv_facts_tools.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+from __future__ import (absolute_import, division, print_function)
+import pytest
+import pprint
+from ansible_collections.arista.cvp.plugins.module_utils.facts_tools import CvFactsTools
+# from ansible_collections.arista.cvp.plugins.module_utils.fields import FIELD_FACTS_DEVICE, FIELD_FACTS_CONFIGLET, FIELD_FACTS_CONTAINER
+from tests.lib import mock
+from tests.data import facts_unit
+from tests.lib.parametrize import generate_list_from_dict
+from tests.lib.helpers import setup_custom_logger
+from tests.lib.utils import generate_test_ids_dict, generate_test_ids_list
+
+LOGGER = setup_custom_logger(__name__)
+
+# ---------------------------------------------------------------------------- #
+#   FIXTURES
+# ---------------------------------------------------------------------------- #
+
+@pytest.fixture
+def cvp_database():
```

Comment:
C.f. #434 if you want to initialise the database with different data for each test case.

Reply:
Will do in a dedicated PR related to testing